src/components/new-conversation-dialog/NewConversationDialog.js

```diff
@@ -25,6 +24,8 @@ const NewConversationDialog = ({
     setOverflowPadding(padding)
   }
 
+  const { hideNewConversation } = props
```

Comment:
Why not keep it in the same place where it was before?

Reply:
discussed and we can leave it like it ist